defaults/main.yaml

```diff
@@ -33,6 +33,9 @@ gh_runner_workspace_path: /var/cache/github-actions-runner
 # complement with the --tag uninstall
 gh_runner_remove_host: false
 
+# If set then the GitHub Actions service will be installed to run as this user.
+gh_runner_service_user: ""
```

Comment:
What about setting the current default value directly to `gh_runner_service_user`? In that case, we wouldn't need to rely on "whether or not set" it would default to the ansible user (current behavior).

If someone wants to implement this role with a different user, then just tweak the default value to another OS user (:

```suggestion
# Run GitHub Actions service will be installed to run as this user.
gh_runner_service_user: "{{ ansible_user_id }}"
```

IMHO it makes the configuration clearer in regards to what `""` translates to, giving better feedback to future devs

Reply:
Yea that is clearer, thanks!